### DISCUSSION TITLE
Two versions of Dark Reader?

### DISCUSSION THREAD

**Discussion by AgainPsychoX** [darkreader/darkreader#9547] [Q&A] [answered]
On my private laptop Dark Reader looks different than on my business laptop... 

Business laptop:
ID: `eimadpbcbfnmbkopoojfekhnkhdbieeh` Version: `4.9.55`
![image](https://user-images.githubusercontent.com/23557839/184306915-aa206e7a-727c-44d7-ac41-f5d06aeef4eb.png)

Private laptop:
ID: `eimadpbcbfnmbkopoojfekhnkhdbieeh` Version: `4.9.55`
![image](https://user-images.githubusercontent.com/23557839/184307256-9ed70467-f3f4-4de1-ba32-19aabf0ecd0c.png)

I guess the one on my private laptop is better, how do I switch to it on the other system?

How do I export settings between those? I can import/export on private laptop, but I can't find the button in the version on the other one...

**Comment by Gusted** [comment, marked as answer]
You have enabled the preview design on your private laptop. You can make the switch on your bussiness computer by following the steps:

-    Click Dark Reader icon.
-    Click Dev tools (in the bottom-right corner).
-    Click Preview new design.
 
 After that you can use the export & import settings under Settings -> Manage settings

**Reply by legalizecannabis** [reply]
I did not know about this option, and was wondering why desktop didn't have custom color options, for MONTHS or so.  I'd say make this option the default, why not, does it have faults that cause the addon to not work as expected?  I'd say the default of not having color options is unexpected, and a usability / accessibility issue.

Nobody, and I mean, absolutely nobody, is going to know to visit the "dev mode" settings, and then click some small square that turns on "preview mode" -- and also, in doing so, will not immediately realize what the changes will be.

I'd also like more than just light or dark mode, and instead, extra custom modes.
